sweep/sweeper.go

```diff
@@ -33,10 +33,16 @@ func (s *Sweeper) CreateSweepTx(
 
 	sweepTx.LockTime = uint32(height)
 
+	sequence := uint32(0)
+	if htlc.Version == swap.HtlcV2 {
+		sequence = 1
```

Comment:
This is used for both the loop out success path and the loop in timeout path. Do both indeed need the sequence 1?

Reply:
Perhaps it is cleaner to also obtain this value from the `htlc` object, so that all spend details are defined in one place.

Reply:
ptal